MeshLib/VtkOGSEnum.cpp

```diff
@@ -92,7 +92,7 @@ int OGSToVtkCellType(CellType ogs)
 			return VTK_PYRAMID;
 		case CellType::PYRAMID13:
 			return VTK_QUADRATIC_PYRAMID;
-		case CellType::INVALID:
+		default:
```

Comment:
hmmm ... i see ... the problem is that Visual Studio gives a warning because there is no default case ...